zerver/lib/test_runner.py

```diff
@@ -252,16 +252,16 @@ def destroy_test_databases(worker_id: Optional[int]=None) -> None:
             # argument to destroy_test_db.
             if worker_id is not None:
                 """Modified from the Django original to """
-                database_id = random_id_range_start + worker_id
-                connection.creation.destroy_test_db(suffix=str(database_id))
+                database_id = random_id_range_start + "_{}".format(worker_id)
```

Comment:
Let's do `{}_{}.format(...)`, it's more readable.  Or better, call a function `get_database_id(worker_id)` that handles this logic so we're not duplicating it.